src/errors/test-run/templates.js

```diff
@@ -61,12 +61,18 @@ export default {
         ${err.getCallsiteMarkup()}
     `),
 
-    [TYPE.clientFunctionExecutionInterruptionError]: err => markup(err, `
+    [TYPE.clientFunctionInterruptedByPageUnloadError]: err => markup(err, `
         <code>${err.instantiationCallsiteName}</code> execution was interrupted by page unload. This problem may appear if you trigger page navigation from <code>${err.instantiationCallsiteName}</code> code.
 
         ${err.getCallsiteMarkup()}
     `),
 
+    [TYPE.clientFunctionInterruptedByDialogError]: err => markup(err, `
+        <code>${err.instantiationCallsiteName}</code> execution was interrupted by a native <code>${err.dialogType}</code> dialog with text "${err.dialogText}". Make sure that <code>${err.instantiationCallsiteName}</code> code does not trigger native dialogs. If this dialog appears as a result of the previous test action, call a "handle..." function that handles this type of dialog after the action.
```

Comment:
correct this according to https://github.com/DevExpress/testcafe/pull/622#issue-162451323